orchestrator-salt/src/main/resources/salt/salt/ambari/server-start.sls

```diff
@@ -18,6 +18,10 @@ ambari-start-postgresql:
   file.managed:
     - source: salt://ambari/systemd/ambari-server.service
 
+{% if not salt['file.directory_exists']('/yarn-private') %}  # FIXME (BUG-92637): must be disabled for YCloud
```

Comment:
Wouldnt it be better at 'roles:ambari_server': in the top file?

Reply:
oh, sorry :) thanks

Reply:
checked it, its working well